src/code/cutscene_camera.c

```diff
@@ -789,7 +790,7 @@ s16 CutsceneCamera_Interp_MultiPointCubic(Vec3f* camPos, f32* camFov, s16* camRo
     return 0;
 }
 
-static f32 sKnots[38];
+f32 sKnots[38];
```

Comment:
If you are going to remove the `static` probably need to give it a more descriptive name.